src/utils/files.ts

```diff
@@ -0,0 +1,33 @@
+/**
+   * Convert byte counts to larger units like KB or MB.
+   *
+   * Includes one trailing digit (when needed).
+   *
+   * @param  {bytes} number
+   * @return {String} Bytes formatted to nearest reasonable number
+   */
+const humanizeByteCount = (bytes: number) => {
```

Comment:
It might be good to introduce this method in the same commit it is used (as written this commit just adds dead code, although of course I assume it's about to get used somewhere)

Reply:
Actually -- rather than write this ourselves, what do you think of using something like https://www.npmjs.com/package/pretty-bytes (11m weekly downloads)